pages/blog/swr-v2.es-ES copy.mdx

```diff
@@ -0,0 +1,267 @@
+---
```

Comment:
need to rename the file, removing ` copy` from filename

Reply:
I've fixed!